src/main/java/androidx/graphics/Path.kt

```diff
@@ -116,3 +116,19 @@ inline infix fun Path.xor(p: Path): Path {
         op(p, Path.Op.XOR)
     }
 }
+
+/**
+ * Generate a closed shape from [points].
+ */
+@RequiresApi(19)
+inline fun Path.fromPoints(points: List<PointF>){
```

Comment:
With a name like `fromPoints` I would expect this to be a factory function but it's a mutator.